### PR TITLE
MAINT: include benchmarks, complete docs, dev tool files in sdist.

### PR DESCRIPTION
**Pull request by rgommers**
lots of stuff was missing ....

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -12,21 +13,29 @@ recursive-include numpy/random/mtrand *.pyx *.pxd
 # Add build support that should go in sdist, but not go in bdist/be installed
 recursive-include numpy/_build_utils *
 recursive-include numpy/linalg/lapack_lite *.c *.h
-include tox.ini
+include runtests.py
+include tox.ini pytest.ini .coveragerc
+recursive-include tools *
+# Cached Cython signatures
+include cythonize.dat
```
Comment (charris) [inline MANIFEST.in]: `cythonize.dat` isn't part of the repo, it is generated and goes away after `git clean -xdf`.

Comment (charris) [reply MANIFEST.in]: That is causing the tests to fail. Does it need to be included in a source distribution?

Comment (rgommers) [reply MANIFEST.in]: I know it's generated, but this is (or was?) the mechanism by which we avoid requiring having Cython installed when building from an sdist. I'll double check.

Comment (charris) [reply MANIFEST.in]: The generated c files are included in the source archives, `python setup.py sdist ` seems to do the job.

Comment (charris) [reply MANIFEST.in]: I'll bet the when people try to build they get an error about needing cython and just pip install it. I cannot see that the cythonize step gets skipped unless pip install from an archive does something special. The `cythonize.dat` file is definitely missing from the archives.

Comment (charris) [reply MANIFEST.in]: Ah, here it is in `setup.py`
```
    if run_build:
        from numpy.distutils.core import setup
        cwd = os.path.abspath(os.path.dirname(__file__))
        if not os.path.exists(os.path.join(cwd, 'PKG-INFO')):
            # Generate Cython sources, unless building from source release
            generate_cython()
```

Comment (rgommers) [reply MANIFEST.in]: ah yes, that rings a bell. removed `cythonize.dat` again

**Comment by charris** [comment]
Thanks Ralf.

**Pull request merged**